platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/WorkflowI18n.java

```diff
@@ -186,6 +186,8 @@ public interface WorkflowI18n extends CoreI18n {
 		.de("Aufgabe delegiert an %s.");
 	I18n0 THE_ITEM_RESIDES_IN_AN_UNEXPECTED_WORKFLOW_NODE = new I18n0("The item resides in an unexpected workflow node.")//
 		.de("Das Element befindet sich in einem unerwarteten Arbeitsablaufknoten.");
+	I18n0 THE_WORKFLOW_ITEM_COULD_NOT_BE_RELOADED = new I18n0("The workflow item could not be reloaded.")//
```

Comment:
This entry is not referenced. I don't think we need it though. See comment on `WorkflowAutoTransitionExecutor.java` below.

Reply:
I removed the entry